README.md

```diff
@@ -78,6 +79,21 @@ To launch the interactive CLI, run the executable without any arguments.
 
 Run `ficsit help` to see a list of available commands.
 
+## Game Installations
+
+The app needs to know where the game is installed. On Steam, this will be
```

Comment:
I thought it could auto-detect some common installs? Or maybe I'm thinking of it picking up on installs that SMM found in the past

Reply:
```suggestion
The app needs to know where the game is installed.
On Steam, this depends on your chosen location, but by default is 
```

Reply:
Superseded by https://github.com/satisfactorymodding/ficsit-cli/pull/61/files#r1562711449